javascript/javascript1/week3/homework/main-1.js

```diff
@@ -0,0 +1,13 @@
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "Kristina",
+  "Rasmus",
+  "Samuel",
+  "Katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+const removedItem = names.splice(names.indexOf("Ahmad"), 1)
```

Comment:
Could you think of a way to make your code more dynamic ? Right now it removes only "Ahmad" and it won't work when the nameToRemove changes :-)

Reply:
Do you mean replace const with let?